plugins/aws/s3/bucketSecureTransportEnabled.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
             // Check the bucket policy
             if (getBucketPolicy && getBucketPolicy.err &&
                 getBucketPolicy.err.code && getBucketPolicy.err.code === 'NoSuchBucketPolicy') {
-                helpers.addResult(results, 2, 'No bucket policy found', 'global', resource);
+                helpers.addResult(results, 0, 'No bucket policy found', 'global', resource);
```

Comment:
Is it okay to not to have any bucket policy? Does that mean the ssl is set by default?

Reply:
no this is wrong, akhtar is fixing now